addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -1,112 +1,130 @@
-"610290","610290","Other admin expenses","expense","False"
-"610300","610300","Accounting","expense","False"
-"610310","610310","Auditing","expense","False"
-"610320","610320","Consultancy","expense","False"
-"610330","610330","Legal and professional charges","expense","False"
-"610340","610340","Exchange gains/losses","expense","False"
-"610350","610350","Other sundry expenses","expense","False"
-"610360","610360","Bad debts","expense","False"
-"610370","610370","Interest paid","expense","False"
-"610380","610380","Bank Charges","expense","False"
-"610390","610390","Donations","expense","False"
-"610400","610400","Entertaining","expense","False"
-"610410","610410","Insurance","expense","False"
-"610420","610420","Travel and subsistence","expense","False"
-"610430","610430","Corporation tax expense","expense","False"
-"610440","610440","Foreign Exchange Gains/Losses","expense","False"
-"610450","610450","Price Differences Control","expense","False"
-"610460","610460","Cash Register Gains/Losses","expense","False"
+"111001","111001","Raw materials on hand","asset_current","True"
+"111002","111002","Stock Work In Progress","asset_current","False"
+"111003","111003","Stock Finished Goods","asset_current","False"
+"111004","111004","Stock Delivered Control","asset_current","True"
+"112001","112001","Trade Receivables","asset_current","True"
+"112002","112002","Debtors Control Account","asset_receivable","True"
+"112003","112003","Debtors Adjustments","asset_current","True"
+"112004","112004","Provision for Doubtful Debts","asset_current","True"
+"112005","112005","Outstanding Receipts","asset_current","True"
+"112006","112006","Debtors Suspense Account","asset_current","False"
+"113001","113001","Main Cheque Acc - XXXXXX","asset_cash","True"
+"113011","113011","Cash Stash / Savings - XXXXXX","asset_cash","False"
+"113021","113021","Credit Card Acc - XXXXX","asset_cash","True"
+"113031","113031","Pay Pal","asset_cash","False"
+"113041","113041","Cash on Hand","asset_cash","False"
+"113042","113042","Cash Control Account","asset_current","True"
+"113051","113051","Bank Suspense Account","asset_current","False"
+"114001","114001","Investment Account - XXXX","asset_current","False"
+"114002","114002","Investment Account - XXXX","asset_current","False"
+"114003","114003","Investment Account - XXXX","asset_current","False"
+"115001","115001","Accrued Interest","asset_current","False"
+"115002","115002","Rental Deposit Paid xxx","asset_current","False"
+"115003","115003","Liquidity Transfer","asset_current","True"
+"121001","121001","Computer & IT Equipment at cost","asset_fixed","False"
+"121002","121002","Computer & IT Equipment Acc Depr","asset_fixed","False"
+"122001","122001","Technical Equipment at cost","asset_fixed","False"
+"122002","122002","Tech Equipment Acc Depr","asset_fixed","False"
+"123101","123101","Motor Vehicle Reg XXX Cost","asset_fixed","False"
+"123102","123102","Motor Vehicle Reg XXX Acc Depr","asset_fixed","False"
```

Comment:
This is certainly a customer specific COA, no need to change everything - just add the missing accounts